main.py

```diff
@@ -75,6 +90,7 @@
         for element in mailbox.newchannels:
             # element.name - name of the channel;
             # element.owner - owner of the channel;
+            # element.members - members to be added to the channel [NEEDS TO BE MADE YET]
```

Comment:
Add this - Won't approve until it works ;P

Reply:
Sounds reasonable to me.